Cargo.toml

```diff
@@ -56,17 +56,24 @@ rand = { version = "0.8.4", optional = true, default-features = false }
 rand_chacha = { version = "0.3.1", optional = true, default-features = false }
 
 [dependencies.noto-sans-mono-bitmap]
-version = "0.1.2"
+path = "../noto-sans-mono-bitmap"
 default-features = false
-features = ["regular", "size_14"]
+features = [
+    "regular",
+    "size_16",
+    "unicode-basic-latin",
+    "unicode-latin-1-supplement",
+    "unicode-latin-extended-a",
```

Comment:
is this needed?

Reply:
I think we only need basic latin. All the log messages and file names shoud be ASCII.

Reply:
alright